src/components/InfoTooltip/TextTooltip.tsx

```diff
@@ -1,17 +1,25 @@
 import React, { useState } from 'react';
-import { StyledSpan, StyledTooltip, StyledCloseIcon } from './Tooltip.style';
+import {
+  StyledTooltip,
+  StyledCloseIcon,
+  SmallTooltipAnchorText,
+} from './Tooltip.style';
 import { StyledTooltipProps } from 'components/InfoTooltip';
 import VisuallyHiddenDiv from 'components/VisuallyHidden/VisuallyHidden';
 import { useBreakpoint } from 'common/hooks';
 import { tooltipAnchorOnClick } from 'components/InfoTooltip';
 import { v4 as uuidv4 } from 'uuid';
 
-const DisclaimerTooltip: React.FC<StyledTooltipProps> = props => {
+const TextTooltip: React.FC<StyledTooltipProps> = props => {
   const [isOpen, setIsOpen] = useState(false);
   const isMobile = useBreakpoint(600);
 
   const idForAccessability = uuidv4();
 
+  const AnchorTextComponent = props.altAnchorComponent
+    ? props.altAnchorComponent
+    : SmallTooltipAnchorText;
```

Comment:
is there a better way? prob. but if/when we start implementing more text tooltips with varying styling, we can pass an anchor text component in every time. this feels good enough for now though